pkg/cvo/cvo_test.go

```diff
@@ -3272,169 +3272,64 @@ func fakeClientsetWithUpdates(obj *configv1.ClusterVersion) *fake.Clientset {
+			wantErr:  true,
 		},
 		{
-			name: "loads valid configuration",
-			update: &payload.Update{
-				Manifests: []lib.Manifest{
-					{
-						GVK: schema.GroupVersionKind{Version: "v1", Kind: "ConfigMap"},
-						Obj: &unstructured.Unstructured{
-							Object: map[string]interface{}{
-								"metadata": map[string]interface{}{
-									"name":      "release-verification",
-									"namespace": "openshift-config-managed",
-									"annotations": map[string]interface{}{
-										"release.openshift.io/verification-config-map": "",
-									},
-								},
-								"data": map[string]interface{}{
-									"verifier-public-key-redhat": string(redhatData),
-									"store-local":                "file://../verify/testdata/signatures",
-								},
-							},
-						},
-					},
-				},
-			},
+			name:          "loads valid configuration",
+			fileName:      "loads-valid.yaml",
+			update:        &payload.Update{},
 			want:          true,
 			wantVerifiers: 1,
 		},
 		{
-			name: "only the first valid configuration is used",
-			update: &payload.Update{
-				Manifests: []lib.Manifest{
-					{
-						GVK: schema.GroupVersionKind{Version: "v1", Kind: "ConfigMap"},
-						Obj: &unstructured.Unstructured{
-							Object: map[string]interface{}{
-								"metadata": map[string]interface{}{
-									"name":      "release-verification",
-									"namespace": "openshift-config-managed",
-									"annotations": map[string]interface{}{
-										"release.openshift.io/verification-config-map": "",
-									},
-								},
-								"data": map[string]interface{}{
-									"verifier-public-key-redhat": string(redhatData),
-									"store-local":                "\nfile://../verify/testdata/signatures\n",
-								},
-							},
-						},
-					},
-					{
-						GVK: schema.GroupVersionKind{Version: "v1", Kind: "ConfigMap"},
-						Obj: &unstructured.Unstructured{
-							Object: map[string]interface{}{
-								"metadata": map[string]interface{}{
-									"name":      "release-verificatio-2n",
-									"namespace": "openshift-config-managed",
-									"annotations": map[string]interface{}{
-										"release.openshift.io/verification-config-map": "",
-									},
-								},
-								"data": map[string]interface{}{
-									"verifier-public-key-redhat":   string(redhatData),
-									"verifier-public-key-redhat-2": string(redhatData),
-									"store-local":                  "file://../verify/testdata/signatures",
-								},
-							},
-						},
-					},
-				},
-			},
+			name:          "only the first valid configuration is used",
+			fileName:      "only-first-used.yaml",
+			update:        &payload.Update{},
 			want:          true,
 			wantVerifiers: 1,
 		},
 	}
 	for _, tt := range tests {
+		raw, err := ioutil.ReadFile(filepath.Join("..", "verify", "testdata", "manifests", tt.fileName))
+		if err != nil {
+			t.Fatal(err)
+		}
+		if tt.fileName != "no-op.yaml" {
+			ms, err := lib.ParseManifests(bytes.NewReader(raw))
```

Comment:
Why are we not parsing `no-op.yaml`?  And if not parsing it makes sense, why are we reading it from disk at all?

Reply:
Good catch. Left over from testing. It contains no manifests, which is the test, so, right, no need for an actual file.